api/src/main/java/com/codeforcommunity/rest/subrouter/ProtectedSiteRouter.java

```diff
@@ -416,4 +417,18 @@ private void handleApproveSiteImage(RoutingContext ctx) {
 
     end(ctx.response(), 200);
   }
+  private void registerGetUnapprovedImages(Router router) {
+    Route unapprovedSiteImages = router.get("/unapproved_images");
+    unapprovedSiteImages.handler(this::handleGetUnapprovedImages);
+  }
+
+  private void handleGetUnapprovedImages(RoutingContext ctx) {
+    JWTData userData = ctx.get("jwt_data");
+    List<SiteEntryImage> images = processor.getUnapprovedImages(userData);
+    List<JsonObject> jsonImages = images.stream()
```

Comment:
⛏️ We don't actually need this step of converting `SiteEntryImages` to `JsonObjects` to return JSON at the end - calling `JsonObject.mapFrom()` takes care of converting regular java objects into JSON for us